src/Rules/PropertyConditions/ElementGroups.cs

```diff
@@ -125,6 +125,7 @@ private static Condition CreateNameRequiredCondition()
             var trueWhenElementHasSiblingsOfSameType = (Header | StatusBar) & SiblingsOfSameType;
             var allowedImage = Image & NotParent(Button | ListItem | MenuItem | TreeItem);
             var allowedText = Text & Condition.Create(HasAllowedPlatformPropertiesForText);
+            var excludedByClassName = WPF & (StringProperties.ClassName.Is("Popup") | StringProperties.ClassName.Is("ContextMenu"));
```

Comment:
If I had my druthers, I'd prefer a name which describes the match like `WPFPopupsAndContextMenus`.

Reply:
Renamed the local variable to `isWPFPopupOrContextMenu`